apps/emqx_bridge/src/schema/emqx_bridge_enterprise.erl

```diff
@@ -32,8 +32,7 @@ api_schemas(Method) ->
         api_ref(emqx_bridge_mongodb, <<"mongodb_rs">>, Method ++ "_rs"),
         api_ref(emqx_bridge_mongodb, <<"mongodb_sharded">>, Method ++ "_sharded"),
         api_ref(emqx_bridge_mongodb, <<"mongodb_single">>, Method ++ "_single"),
-        %% TODO: un-hide for e5.2.0...
-        %%api_ref(emqx_bridge_hstreamdb, <<"hstreamdb">>, Method),
+        api_ref(emqx_bridge_hstreamdb, <<"hstreamdb">>, Method),
```

Comment:
Since this was just uncommented, I suggest adding a test case to ensure that the bridge probe HTTP API works properly, as it often behaves a little different and may break while the bridge create/update API works.  Adding a `emqx_bridge_testlib:t_start_stop/2` case should exercise that path.